carbonmark/pages/api/certificates/[beneficiaryAddress]/[retirementIndex]/index.ts

```diff
@@ -0,0 +1,68 @@
+import { utils } from "ethers";
+import isNumber from "lodash/isNumber";
+import { NextApiRequest, NextApiResponse } from "next/types";
+
+import { urls } from "@klimadao/lib/constants";
+import {
+  getRetirementTokenByAddress,
+  queryKlimaRetireByIndex,
+} from "@klimadao/lib/utils";
+
+import { IS_PRODUCTION } from "lib/constants";
+import { generateCertificate } from "lib/retirementCertificates";
+
+type Query = {
+  beneficiaryAddress: string;
+  retirementIndex: string;
+};
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (IS_PRODUCTION) {
+    return res.status(404).send("Not found");
+  }
+
+  try {
+    const { beneficiaryAddress, retirementIndex } = req.query as Query;
+
+    /** Validate address params */
+    if (
+      !beneficiaryAddress ||
+      !retirementIndex ||
+      !isNumber(Number(retirementIndex))
+    ) {
+      return res.status(404).send("Not found");
+    }
+
+    if (!utils.isAddress(beneficiaryAddress)) {
+      return res.status(400).send("Invalid beneficiary address");
+    }
+
+    /** Retirement indexes start from 0 starts from 1 */
+    const index = Number(retirementIndex) - 1;
+    const retirement = await queryKlimaRetireByIndex(beneficiaryAddress, index);
+
+    /** Validate fetched data */
+    if (!retirement) {
+      return res.status(404).send("Not found");
+    }
+
+    const certificateParams = {
+      retirement,
+      retirementIndex,
+      retirementUrl: `${urls.retirements}/${retirement.beneficiaryAddress}/${retirementIndex}`,
+      retiredToken: getRetirementTokenByAddress(retirement.pool),
+    };
+
+    res.setHeader("Content-Type", "application/pdf");
+    res.setHeader("Cache-Control", "max-age=86400");
```

Comment:
Looks like the vercel CDN cache is not picking this up, try changing this to s-max-age
```suggestion
    res.setHeader("Cache-Control", "s-max-age=86400");
```

![image](https://user-images.githubusercontent.com/88635679/234430778-f4f805f6-aad8-43f1-8c99-2833ad62f6ba.png)

Reply:
fixed that one up, used `s-maxage=n` as referenced in the [docs](https://vercel.com/docs/concepts/edge-network/caching#serverless-and-edge-functions)